### PR TITLE
Adding cdk context instructions to multi mon

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*
Adding cdk context instructions to multi mon

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/patterns/multi-account-monitoring.md [hunk 1]
```diff
@@ -44,7 +44,31 @@ You can find the team-geordie configuration for this pattern in the workload rep
...
+            "region": "your_region"
+        },
+        "monitoringEnv": {
+            "account": "444444444444",
+            "region": "your_region"
+        }
+    }
+    }
+    ```
+7. Create the follwing IAM users and attach `administrator` policy to required accounts.
```
Comment (shapirov103) [inline docs/patterns/multi-account-monitoring.md:71]: minor: follwing => following

File: docs/patterns/multi-account-monitoring.md [hunk 2]
```diff
@@ -44,7 +44,31 @@ You can find the team-geordie configuration for this pattern in the workload rep
    1. *repo* - to read the repository
    2. *admin:repo_hook* - if you plan to use webhooks (enabled by default)
 
-6. Create the follwing IAM users and attach `administrator` policy to required accounts.
+6. `cdk-context` secret secret must be stored as a plain text in the following format in AWS Secrets Manager for cdk context for all the 4 AWS accounts used by the solution.
+
+    ```
+    {
+    "context": {
+        "prodEnv1": {
```
Comment (shapirov103) [inline docs/patterns/multi-account-monitoring.md:52]: Will it be clear to the customers what each account is for? I suppose adding a short blurb that prodEnv1 and 2 are workload accounts, monitoring is aggregated monitoring from all prod accounts and pipeline is CICD (which is separate as a separate DevOps team can maintain it).

File: docs/patterns/multi-account-monitoring.md [hunk 3]
```diff
@@ -53,19 +77,19 @@ You can find the team-geordie configuration for this pattern in the workload rep
...
 
-9. Modify the code in your forked repo to point to your GitHub username/organisation. This is needed because the AWS CodePipeline that will be automatically created will be triggered upon commits that are made in your forked repo. Open the [pattern file source code](../../lib/pipeline-multi-env-gitops/index.ts) and look for the declared const of `gitOwner`. Change it to your GitHub username.
+10. Modify the code in your forked repo to point to your GitHub username/organisation. This is needed because the AWS CodePipeline that will be automatically created will be triggered upon commits that are made in your forked repo. Open the [pattern file source code](../../lib/pipeline-multi-env-gitops/index.ts) and look for the declared const of `gitOwner`. Change it to your GitHub username.
 
-10. Once all pre-requisites are set you are ready to deploy the pipeline. Run the following command from the root of this repository to deploy the pipeline stack:
+11. Once all pre-requisites are set you are ready to deploy the pipeline. Run the following command from the root of this repository to deploy the pipeline stack:
 
 ```bash
 npx cdk multi-account-central-pipeline
```
Comment (shapirov103) [inline docs/patterns/multi-account-monitoring.md:89]: I assume you meant `npx cdk deploy` here.

with the latest change can you add:
```
make build # or npm run build
npx cdk deploy mulit-account-central-pipeline
```

build will be mandatory (but cdk deploy is twice as fast)

**Review by shapirov103 — COMMENTED** [review]
Looks great, a couple of small comments.

File: docs/patterns/multi-account-monitoring.md (hunk 1 above)
Comment (elamaran11) [reply docs/patterns/multi-account-monitoring.md]: Fixed.

File: docs/patterns/multi-account-monitoring.md (hunk 3 above)
Comment (elamaran11) [reply docs/patterns/multi-account-monitoring.md]: This is also fixed.

**Comment by elamaran11** [comment]
@shapirov103 Addressed all your comments.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**